api/types/client.go

```diff
@@ -93,8 +93,9 @@ type SiteConfigSpec struct {
 	SiteTtl             time.Duration
 	EnableConsole       bool
 	EnableFlowCollector bool
+	FlowRecordTtl       time.Duration
```

Comment:
Could put this under FlowControllerOptions, but that is very minor.

Reply:
agree, will update